internal/configs/config.textformats.go

```diff
@@ -1,13 +1,18 @@
 package configs
 
-import "strings"
+import (
+	"strings"
+)
 
 type TextFormats struct {
-	Prompt                  ConfigString `yaml:"Prompt"`                  // The in-game status prompt style
-	EnterRoomMessageWrapper ConfigString `yaml:"EnterRoomMessageWrapper"` // Special enter messages
-	ExitRoomMessageWrapper  ConfigString `yaml:"ExitRoomMessageWrapper"`  // Special exit messages
-	Time                    ConfigString `yaml:"Time"`                    // How to format time when displaying real time
-	TimeShort               ConfigString `yaml:"TimeShort"`               // How to format time when displaying real time (shortform)
+	Prompt                  ConfigString      `yaml:"Prompt"`                  // The in-game status prompt style
+	EnterRoomMessageWrapper ConfigString      `yaml:"EnterRoomMessageWrapper"` // Special enter messages
+	ExitRoomMessageWrapper  ConfigString      `yaml:"ExitRoomMessageWrapper"`  // Special exit messages
+	Time                    ConfigString      `yaml:"Time"`                    // How to format time when displaying real time
+	TimeShort               ConfigString      `yaml:"TimeShort"`               // How to format time when displaying real time (shortform)
+	DefaultLanguage         ConfigString      `yaml:"DefaultLanguage"`         // Specify the default game language (fallback)
+	Language                ConfigString      `yaml:"Language"`                // Specify the game language
+	LanguagePaths           ConfigSliceString `yaml:"LanguagePaths"`           // Specify the game language file paths
```

Comment:
I think we should either:

1. Have a Translation struct all its own in the Config that contains all 3 of these
or
2. Put the LanguagePaths in the `Config.FilePaths` struct

Reply:
Preferred option 1, will update later.